odl/space/ntuples.py

```diff
@@ -1426,9 +1461,11 @@ def weighted_dist(weight, use_inner=False):
         evaluate to exactly zero for equal (but not identical)
         `x` and `y`.
 
+        Can only be used if `exponent` is 2.0.
+
     Returns
     -------
-    inner : callable
+    dist : callable
```

Comment:
I've been doing this on the doc branch, but anytime you write callable, you can do `callable`, that way you get a link to the python doc, in case someone doesnt underdand.